test/go-tests/test_utils.go

```diff
@@ -179,11 +179,17 @@ func GetInternalKeptnAPI(ctx context.Context, internalService, localPort string,
 	if err != nil {
 		return nil, err
 	}
-	keptnInternalAPI, err := NewAPICallerWithBaseURL("http://127.0.0.1:" + localPort)
+	var internalKeptnAPI *APICaller
+
+	err = retry.Retry(func() error {
+		internalKeptnAPI, err = NewAPICallerWithBaseURL("http://127.0.0.1:" + localPort)
+		return err
+	}, retry.NumberOfRetries(5), retry.DelayBetweenRetries(1*time.Minute))
```

Comment:
1 Minute? That's quite a big delay between retries. Wouldn't it make sense to lower that in the range of several seconds?

Reply:
tested multiple iterations of this, in all cluster it works without even retry logic but k3d 24 fails all the time even with 40sec :(